<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="drawingArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>750</width>
      <height>750</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Initial X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>100</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Final X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>60</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Initial Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>140</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Final Y:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="mInitialXEditText">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="mFinalYEditText">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>140</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="mFinalXEditText">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>100</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="mInitialYEditText">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>60</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="mDrawThinLine">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>250</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Thin Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mDrawDottedLine">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>290</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Dotted Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mDrawDashedDottedLine">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>410</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Dashed-Dotted Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mDrawDashedLine">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>370</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Dashed Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mDrawThickLine">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>330</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Thick Line</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="mWidthEditText">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>190</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>190</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Width:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
